packages/admin/admin-icons/src/CometColor.tsx

```diff
@@ -1,23 +1,31 @@
 import { SvgIcon, SvgIconProps } from "@mui/material";
 
+function getUniqueId() {
+    return Math.random().toString(36).substring(7);
+}
+
 export default function Comet(props: SvgIconProps) {
+    const uniqueIdA = getUniqueId();
+    const uniqueIdB = getUniqueId();
+    const uniqueIdC = getUniqueId();
```

Comment:
This changes for every render of the icon. You'll probably need something like [useConstant](https://github.com/vivid-planet/comet/blob/improve-pull-request-template/packages/admin/admin/src/router/ForcePromptRoute.tsx#L9)

Reply:
I just moved it out of the component. That should also be fine right? https://github.com/vivid-planet/comet/pull/2529/commits/78374344f7a03ea990abaa24a322840c817f97d9